README.rst

```diff
@@ -5,6 +5,62 @@ This repository contains kiwi_ definitions to build containers for
 `SUSE OpenStack Cloud`_.
 The containers are built in the `Open Build Service`_.
 
+Generating the .kiwi files from templates
+=========================================
+
+We want to build containers based on different base containers (eg. for
+openSUSE or SLE). To accomplish that, we use Jinja2 templates and
+YAML based (.yml) files which contain variables to fill the templates.
+
+The tool to combine the single template (usually .kiwi.j2) and the
+variable files(s) (usually .yml) is `j2gen`_
```

Comment:
While not strictly wrong here I'd prefer to use "process" instead of "combine" here. But feel free to disagree :)

Reply:
Done